index.js

```
@@ -170,6 +170,11 @@ const createClient = (profile, userAgent, request = _request) => {
 			filters.push(profile.filters.accessibility[opt.accessibility])
 		}
 
+		const gisFltrL = [{
+			meta: ['slow','normal','fast'].includes(opt.walkingSpeed) ? 'foot_speed_' + opt.walkingSpeed : 'foot_speed_normal',
```

Sorry, I didn't explain clearly enough. Please throw on invalid usage, as seen above.